package.json

```diff
@@ -221,7 +221,7 @@
   "resolutionsPurpose": "This temporarily chooses the right versions of these libs, which are specified in both deck AND deck-kayenta",
   "resolutions": {
     "@uirouter/react-hybrid": "1.0.0",
```

Comment:
Update all 4 of these resolutions, and also update the versions in the `"dependencies"` block